discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/RegistryService.java

```diff
@@ -95,6 +95,19 @@ public interface RegistryService<T> {
      */
     List<InetSocketAddress> lookup(String key) throws Exception;
 
+    /**
+     * Get all nodes' ip in that server's cluster
+     * @return ips
+     * @throws Exception the exception
+     */
+    List<InetSocketAddress> getClusterNodes() throws Exception;
```

Comment:
Looking at the various implementations, this interface I don't think is necessary.

Reply:
Although the core logic in `getClusterNodes()` is calling the private method `lookupByCluster`, which the same as an  existing function `lookup`, but `lookup` is exposed to be used in CLIENT side.

So IMHO, we need expose a method to be used in SERVER side. That's why I introduce the `getClusterNodes`.

Reply:
ok.